### PR TITLE
Fluid Typography #41

### LINKED ISSUE #41: Fluid Typography
Change font size fluidly based on screen width.

https://css-tricks.com/simplified-fluid-typography/

### PR DESCRIPTION
**Pull request by arumsey**
## Issue

Fix #41

## 🔗 Test URLs:

- Before: https://main--sgws-creative--hlxsites.hlx.page/bloominbrands/p/1
- After: https://fluid-fonts--sgws-creative--hlxsites.hlx.page/bloominbrands/p/1

## 📝 Description:

- use relative values for font sizes

### CODE REVIEW

**Review by tmathern — COMMENTED** [review]
If the arrow icons are here, should they also scale? (On Chome, for me, they did not)

File: styles/styles.css [hunk 1]
```diff
@@ -58,10 +57,11 @@
 }
 
 body {
-  font-size: var(--body-font-size-s);
+  font-size: var(--body-font-size);
+  font-size: clamp(var(--body-font-size-min), 0.75vw + 0.3rem, var(--body-font-size-max));
   margin: 0;
   font-family: var(--body-font-family);
-  line-height: 1.6;
+  line-height: 1.75;
```
[the remark this answers is not in the document]
Comment (arumsey) [reply styles/styles.css:64]: Line height is relative to font-size.

**Comment by arumsey** [comment]
> The titles should scale too

I've added scaling to `h1`, `h2`, and `h3` so far to see where they gets us.

**Comment by tmathern** [comment]
> > The titles should scale too
> 
> I've added scaling to `h1`, `h2`, and `h3` so far to see where they gets us.

Then it doesn't work for me on Chrome - like the stats (h2) don't scale. Or is it limited to some blocks/sections for now?

**Comment by arumsey** [comment]
> If the arrow icons are here, should they also scale? (On Chome, for me, they did not)

Icons within buttons won't scale yet but inline icons should scale with they font size.

**Comment by tmathern** [comment]
> > If the arrow icons are here, should they also scale? (On Chome, for me, they did not)
> 
> Icons within buttons won't scale yet but inline icons should scale with they font size.

Yes, those icons work!

**Comment by arumsey** [comment]
> 

TabView is overwriting the default font-size since ti comes from a theme. Let me see what can be done about that.

**Review by tmathern — APPROVED** [review]
LGTM

**Pull request merged**